src/components/Forms/Select.js

```diff
@@ -0,0 +1,351 @@
+      floatingLabelText
+    } = this.props
+
+    const { isOpen, selectedOption, isFocused } = this.state
+    const hasValue = selectedOption !== null
+    const showHintText = hintText && !hasValue && isOpen
+
+    return (
+      <div
+        onFocus={this._handleFocus}
+        onBlur={this._handleBlur}
+      >
+        <div
+          style={[
+            styles.trigger,
+            disabled && styles.triggerDisabled,
+            (isOpen || isFocused) && !hasError && styles.highlight,
+            !disabled && hasError && styles.error
+          ]}
+          aria-required={required}
+          aria-invalid={hasError}
+          tabIndex={0}
+          ref={ (node) => this.trigger = node }
+        >
+          <FloatingLabel
+            text={floatingLabelText}
+            float={isOpen || hasValue}
+            disabled={disabled}
+            isActive={isOpen || isFocused}
+            hasError={hasError}
+            style={styles.floatingLabel}
+          />
+
+          <div style={styles.labelContainer}>
+            { hintText &&
+              <TextFieldHint
+                text={hintText}
+                show={showHintText}
+                disabled={disabled}
+                style={styles.floatingLabel}
+              />
+            }
+
+            {selectedOption && selectedOption.label}
+          </div>
+
+          <div style={styles.iconContainer}>
+            <Icon
+              name="arrowDownSmallBold"
+              style={[
+                styles.icon,
+                isOpen && styles.iconOpen,
+                disabled && styles.iconDisabled,
+                !disabled && hasError && styles.iconError
+              ]}
+            />
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  render() {
+    const {
+      children,
+      disabled,
+      id,
+      isValid,
+      halfWidth,
+      fullWidth,
+      onOpen,
+      serverError,
+      style,
+      validationErrorText
+    } = this.props
+
+    const { isOpen } = this.state
+
+    return (
+      <div
+        style={[
+          styles.root,
+          fullWidth && styles.fullWidth,
+          halfWidth && styles.halfWidth,
+          style
+        ]}
+      >
+        <div>
+          {serverError && !disabled && !isValid &&
+            <ServerError
+              text={serverError}
+            />
+          }
+
+          <DropdownMenu
+            triggerElement={this.renderTrigger()}
+            open={isOpen}
+            onRequestChange={this._handleRequestChange}
+            onSelect={this._handleSelect}
+            menuProps={{style: styles.menu}}
+            onClose={this._handleClose}
+            onOpen={onOpen}
+          >
+            {children}
+          </DropdownMenu>
+
+          <ValidationError
+            text={validationErrorText}
+            show={!disabled && !isValid && !serverError}
+            inputId={id}
+          />
+        </div>
+      </div>
+    )
+  }
+}
+
+export default Select
```

Comment:
Hovering on the text in the select gives me a cursor rather than a pointer
![screen shot 2017-12-04 at 3 30 49 pm](https://user-images.githubusercontent.com/1747661/33582056-30e47b18-d908-11e7-9c42-32f4dff5b09d.png)